en/modifier-type.json

```
@@ -149,7 +149,7 @@
     "RARER_CANDY": { "name": "Rarer Candy" },
 
     "MEGA_BRACELET": { "name": "Mega Bracelet", "description": "Mega Stones become available." },
-    "DYNAMAX_BAND": { "name": "Dynamax Band", "description": "Max Mushrooms become available." },
+    "DYNAMAX_BAND": { "name": "Dynamax Band", "description": "Max Mushrooms become available. Dynamax Pokemon take 1/3 less damage." },
```

```suggestion
    "DYNAMAX_BAND": { "name": "Dynamax Band", "description": "Max Mushrooms become available. This lets you Dynamax your Pokémon, which makes it take 1/3 less damage." },
```
(Trying to reword it so that it doesn't sound like an additional effect of the Dynamax Band itself)